extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -18,6 +18,18 @@
 	'portable-infobox-xml-parse-error-tag-not-finished' => 'Premature end of tag',
 	'portable-infobox-xml-parse-error-infobox-tag-attribute-unsupported' => 'Attribute "$1" is not supported in
 	<infobox> tag',
+
+	'portable-infobox-builder-tite' => 'Portable Infobox Builder',
+	'portable-infobox-builder-no-template-title-set' => 'No template title set. Please specify the title for new
+	infobox template.',
+	'portable-infobox-builder-create-template-entry-point-create-infobox' => 'Create Infobox',
+	'portable-infobox-builder-create-template-entry-point-create-regular-template' => 'Create Regular Template',
+
+	'portable-infobox-builder-entry-point-modal-title' => 'Do you want to create an infobox?',
+	'portable-infobox-builder-entry-point-modal-message' => 'Perhaps you are creating a new infobox template? Try out
```

Comment:
In general, unless the linebreaks are intentional, I'd suggest to avoid it as it may affect formatting when parsed